packages/shopify-codemod/test/transforms/function-to-arrow.test.js

```diff
@@ -0,0 +1,12 @@
+import 'test-helper';
+import ternaryStatementToIfStatement from 'function-to-arrow';
+
+describe('functionToArrow', () => {
+  it('transforms the simple case', () => {
+    expect(ternaryStatementToIfStatement).to.transform('function-to-arrow/basic');
+  });
+
+  it("doesn't transform function that use `this`", () => {
+    expect(ternaryStatementToIfStatement).to.transform('function-to-arrow/this');
+  });
```

Comment:
Can you add a function that it still converts a function that has a nested `this` in a callback? Something like:

```js
const foo = function() {
  bar(function() {
    this.baz;
  });
}
```

should become:

```js
const foo = () => {
  bar(function() {
    this.baz;
  });
}
```